wagtail_localize/locales/wagtail_hooks.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import Permission
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from wagtail.admin.menu import MenuItem
+from wagtail.core import hooks
+from wagtail.core.permissions import site_permission_policy
+
+from .views import LocaleViewSet
+
+
+@hooks.register('register_admin_viewset')
```

Comment:
If this hook is in core, is there any way I could convince you to add this to the Wagtail docs?

Reply:
I think the plan was to develop it a bit more into an official API first. That was a few years ago!

Hopefully we will have a chance to revisit this in the platform sub-team